cirq-core/cirq/ops/moment.py

```diff
@@ -49,6 +49,18 @@ def _default_breakdown(qid: 'cirq.Qid') -> Tuple[Any, Any]:
         return None, qid
 
 
+def _have_equal_gates(op1, op2):
+    """Compares two operations and returns a boolean of it their gates
```

Comment:
Prefer a single line summary at the beginning.  Can be changed to:

"""Compares two operations and returns True if their gates match."""

Reply:
Done.